.github/workflows/build_e2e_test_artifacts.yml

```diff
@@ -122,9 +130,8 @@ jobs:
           # MLIR files and MLIR source models).
           # Not archiving the directory to allow fetching each file as needed
           # separately.
-          find "${E2E_TEST_ARTIFACTS_DIR}" -maxdepth 1 \
-            -name "iree_*" -o -name "model_*.mlir" -o -name "model_*.mlirbc" -o -name "*.json" | \
-            gcloud storage cp --read-paths-from-stdin -r \
+          find "${E2E_TEST_ARTIFACTS_DIR}" -maxdepth 1 ! -name "*.tflite" \
```

Comment:
As we no longer import TF models, `*.tflite` is only the source models we want to exclude from the upload.